docs/content/SegmentedControl.mdx

```diff
@@ -151,7 +151,15 @@ description: Use a segmented control to let users select an option from a short
   <PropsTableRow name="aria-label" type="string" />
   <PropsTableRow name="aria-labelledby" type="string" />
   <PropsTableRow name="aria-describedby" type="string" />
-  <PropsTableRow name="fullWidth" type="boolean" description="Whether the control fills the width of its parent" />
+  <PropsTableRow
+    name="fullWidth"
+    type="'boolean' | {
+      narrow?: boolean
+      regular?: boolean
+      width?: boolean
+  }"
```

Comment:
This will make it render a little more nicely in the docs:
```suggestion
    type={`| boolean
| {
  narrow?: boolean
  regular?: boolean
  wide?: boolean
}`}
```